src/pages/saksbehandling/behandlingsoversikt/ferdigeBehandlinger/FerdigeBehandlinger.tsx

```diff
@@ -0,0 +1,90 @@
+import * as RemoteData from '@devexperts/remote-data-ts';
+import { Loader } from '@navikt/ds-react';
+import React, { useEffect, useState } from 'react';
+
+import ApiErrorAlert from '~components/apiErrorAlert/ApiErrorAlert';
+import * as sakSlice from '~features/saksoversikt/sak.slice';
+import { pipe } from '~lib/fp';
+import { useAsyncActionCreator } from '~lib/hooks';
+import { Nullable } from '~lib/types';
+import {
+    Filter,
+    hentFiltrerteVerdier,
+    RestansResultatFilter,
+    RestansTypeFilter,
+} from '~pages/saksbehandling/behandlingsoversikt/filter/Filter';
+import RestanserTabell from '~pages/saksbehandling/restans/Restanser';
+import { Restans, RestansStatus, RestansType } from '~types/Restans';
+import { toDateOrNull } from '~utils/date/dateUtils';
+
+export const FerdigeBehandlinger = () => {
+    const [hentFerdigeBehandlingerStatus, hentFerdigeBehandlinger] = useAsyncActionCreator(
+        sakSlice.hentFerdigeBehandlinger
+    );
+
+    useEffect(() => {
+        hentFerdigeBehandlinger();
+    }, []);
+
+    const tilOgMed = useState<Nullable<Date>>(null);
+    const fraOgMed = useState<Nullable<Date>>(null);
+
+    const [type, setType] = useState<RestansTypeFilter>({
+        [RestansType.SØKNADSBEHANDLING]: false,
+        [RestansType.REVURDERING]: false,
+        [RestansType.KLAGE]: false,
+    });
+
+    const [resultat, setResultat] = useState<RestansResultatFilter>({
+        [RestansStatus.OPPHØR]: false,
+        [RestansStatus.AVSLAG]: false,
+        [RestansStatus.INGEN_ENDRING]: false,
+        [RestansStatus.INNVILGET]: false,
+        [RestansStatus.AVSLUTTET]: false,
+    });
+
+    const filterRestanser = (restanser: Restans[]): Restans[] => {
```

Comment:
```suggestion
    const filtrerRestanser = (restanser: Restans[]): Restans[] => {
```
Litt rart å lese engelsknorske ord 🤷